RestApi/OData/FilterTranslator.g4

```diff
@@ -10,13 +10,12 @@ lexer grammar FilterTranslator;
 	public FilterTranslator(ICharStream input,
 							SqlServerRestApi.TableSpec tableSpec,
 							SqlServerRestApi.QuerySpec querySpec,
-							string odataHelperSqlSchema = "odata"): base(input) 
+							string odataHelperSqlSchema = "odata"): this(input) 
```

Comment:
:memo: Switching from `base` to `this` allowed the custom constructor to leverage existing code for creating the interpreter instance.

Reply:
Thanks, I didn't tried this but this is better solution.